articles/121_realtime_proposal.md

```diff
@@ -0,0 +1,230 @@
+---
+layout: default
+title: Proposal for Implementation of Real-time Systems in ROS 2
+permalink: articles/realtime_proposal.html
+abstract: This article proposes an approach to the real-time performance requirement in ROS 2.
+published: true 
+author: Jackie Kay
+---
+
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+## Requirements and Implementation of Real-Time Systems
+
+### System requirements
+
+The requirements of a real-time system varies depending on the use case.
```

Comment:
`requirements .. varies`

Isn't it either `requirements vary` or `requirement varies` (I think you want the first one)?